### DISCUSSION TITLE
Login feature?

### DISCUSSION THREAD

**Discussion by 7v5** [thenick775/gbajs3#70] [General]
Is there any way to sign up?

**Comment by thenick775** [comment]
Not at this time, unless you know me personally or want to host your own instance yourself.

I don't have the storage capacity to make that public for everyone else at this time, as rom file sizes can stack up quickly.

I do plan on making an account registration feature instead of just entering people into the admin interface yourself, along with s3 backed file storage, but for my publicly hosted page I most likely won't be supporting that on my own hardware.

The account registration feature is still decently far out on the roadmap, as I'll have to figure out how I want to handle emails. But feel free to make a pr if your inclined to help get that started! 😎

**Comment by thenick775** [comment]
Good news though, using the File System modal after loading/saving games/save states, you can persist all roms/saves/save states/cheats to your device's indexed db storage, with no need for an account!

This works best when installed as a PWA, but even that is not required.

In essence, all features available through the profile menu are available on your local device now.

**Reply by 7v5** [reply]
thanks for the info!
